script/axe-docs

```diff
@@ -37,26 +37,31 @@ containsElement () {
 # DO NOT ADD TO THIS LIST!
```

Comment:
I added a few more 😇. But this PR only splits up the existing docs and doesn't add any new components.